packages/edit-post/src/store/actions.js

```diff
@@ -196,6 +196,13 @@ export function updatePreferredStyleVariations( blockName, blockStyle ) {
 	};
 }
 
+/**
+ * Returns an action object used in signalling that the editor should attempt
+ * to locally autosave the current post every `interval` seconds.
+ *
+ * @param {number} interval The new interval, in seconds.
+ * @return {Object} Action object.
+ */
```

Comment:
This won't actually be documented in the data handbook as long as it's experimental:

https://developer.wordpress.org/block-editor/data/data-core-editor/#actions

If we want to document it, are we also wanting to make it stable?

Related: #17743

Reply:
I came just for the documentation, regardless of its experimental status, but was wondering the same. On one hand, it's been a few months since the introduction of this API and it hasn't required any changes. On the other hand, it's arguably a very isolated piece that seldom receives attention, so its lack of change isn't necessarily indicative of soundness or stability.

For instance, I'm wondering whether this functionality might be better provided by something else, whether that's something generic to more editor settings, whether there's room for it in the recently [proposed Features API](https://make.wordpress.org/core/2020/01/23/controlling-the-block-editor/), etc. WDYT?

Reply:
Admittedly, I still haven't got around to digest the implementation on the whole, so I can't really speak to specifics about whether it's something we want to commit to. If you're already considering options for potential reimplementations, then I think it would be fair to keep it experimental. Or, at least, it shouldn't necessarily block this pull request, which is an improvement in its own right.

I'm also mindful of our human aversions to committing to anything as final and, given the choice, we'd make everything "experimental" forever. So, there's that ¯\\_(ツ)\_/¯ . It's always a balance.